docs/install.md

```diff
@@ -261,9 +261,11 @@ each.
 
 ```python
 workspace(
-    name = "my_wksp",
-    managed_directories = {"@app1_npm": ["app1/node_modules"],
-                           "@app2_npm": ["app2/node_modules"]}},
+    name = "maglev_ui",
```

Comment:
you did more than just fix the syntax error - you also changed the names of the repositories, now they don't match the *_install rules below

Reply:
Fixed

Reply:
sorry, it isn't fixed - the names of the repos in managed_directories needs to match the names passed to the yarn_install and npm_install rules below.

just remove the spurious extra `}` in your PR, and optionally format it the way buildifier does

Reply:
I force-pushed a fix to your branch, will merge now. Thanks for pointing out the problem!